code/game/mecha/mecha.dm

```diff
@@ -411,6 +411,10 @@
 		var/lights_energy_drain = 2
 		use_power(lights_energy_drain)
 
+	if(!enclosed && occupant?.incapacitated()) //no sides mean it's easy to just sorta fall out if you're incapacitated.
+		visible_message("<span class='warning'>[occupant] tumbles out of the cockpit!</span>", "<span class='danger'>You tumble out of the cockpit!</span>")

```

Comment:
The second message will never show since src is mecha here.

Reply:
Yeah, I thought I had changed that to just a single message. I'll fix it when I have a moment.

Reply:
Updated